x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/support/commands.ts

```diff
@@ -88,18 +88,14 @@ Cypress.Commands.add('selectAbsoluteTimeRange', (start: string, end: string) =>
   const format = 'MMM D, YYYY @ HH:mm:ss.SSS';
 
   cy.getByTestSubj('superDatePickerstartDatePopoverButton').click();
-  cy.contains('Start date')
-    .nextAll()
-    .find('[data-test-subj="superDatePickerAbsoluteDateInput"]')
-    .clear({ force: true })
+  cy.getByTestSubj('superDatePickerAbsoluteDateInput').clear({ force: true });
+  cy.getByTestSubj('superDatePickerAbsoluteDateInput')
```

Comment:
> [!IMPORTANT]
> This element query is not chained to prevent incurring this [detached reference error](https://docs.cypress.io/guides/references/error-messages#cy-failed-because-the-page-updated).